### ISSUE TITLE
Error when streams tx

### ISSUE THREAD

**Issue by nigeldelviero** [merkle3/merkle-sdk-ts#1]
I want to stream tx, I followed like on the docs but it has different and error at on() listeners:

```typescript
merkle.transactions.stream().on('transaction', tx => {
    console.log('tx from: ', tx.from)
}).on('error', console.log)
```

Error: Expected 1 arguments, but got 2

**Comment by nigeldelviero** [comment]
For now it will works if like this:
```typescript
 merkle.transactions.stream().on((tx: any) => {
   console.log('tx from: ', tx.from)
 });
```

But, if write code like this: 

```typescript
 merkle.transactions.stream().on((tx: any) => {
   console.log(tx)
 });
```

it will print : `Transaction {}`

**Comment by Theo-** [comment]
Thanks! The readme was updated to fix this issue in 276dfb48be2ab75a248f0946cb9adadb7b1467cc
